libavfilter/dnn/dnn_backend_tf.c

```diff
@@ -152,8 +152,8 @@ static DNNReturnType get_input_tf(void *model, DNNData *input, const char *input
     }
     TF_DeleteStatus(status);
 
-    // currently only NHWC is supported
-    av_assert0(dims[0] == 1);
+    //currently only NHWC is supported
+    av_assert0(dims[0] == 1 || dims[0] == -1);
```

Comment:
I forget if this change would be useful upstream...But, in case it is we could isolate it in its own commit.

Reply:
Ahh just saw that somebody beat us to it! 

https://github.com/FFmpeg/FFmpeg/commit/1b1064054c8f3b4ad3b52d14f0c8ee1c4e8200fd

It's currently a part of `master`, so this change should automatically disappear from this commit probably whenever we rebase next and apply our commits on-top.